client/src/components/pages/Players.js

```diff
@@ -343,6 +347,46 @@ export default function Players({ tourney, user }) {
     }
   };
 
+  let isCountryBasedTeamTourney = false;
+  if (!flags.has("suiji") && !flags.has("registerAsTeam")) {
+    isCountryBasedTeamTourney = hasTeams;
+  }
+
+  const countPotentialTeams = () => {
+    let countryPlayerCounts = {};
+    players.forEach(player => {
+      if (countryPlayerCounts[player.country]) {
+        countryPlayerCounts[player.country] += 1;
+      }
+      else {
+        countryPlayerCounts[player.country] = 1;
+      }
+    });
+
+    let minPotentialTeams = 0;
+    let maxPotentialTeams = 0;
+
+    Object.keys(countryPlayerCounts).forEach(k => {
+      if (countryPlayerCounts[k] >= 2) {
+        minPotentialTeams += 1;
+        maxPotentialTeams += Math.min(Math.floor(countryPlayerCounts[k] / 7), 1) + 1;
```

Comment:
so this count 3 teams if a country has 14 players?
maybe should confirm with bries if this is desirable, since i dont think any gts tourneys allow C teams etc.

Reply:
Fairly sure we will never allow C teams so 14 players should count as 2 teams

Reply:
Oops I just assumed that, will fix in a future commit

Reply:
Fixed in 28cbdf7f7d1d074f853160bcff8c0669ac6a7fc6